docs/getting-started-guides/rkt/index.md

```diff
@@ -10,25 +10,30 @@ We still have [a bunch of work](http://issue.k8s.io/8262) to do to make the expe
   *(Note that systemd is not required by rkt itself, we are using it here to monitor and manage the pods launched by kubelet.)*
 
 - Install the latest rkt release according to the instructions [here](https://github.com/coreos/rkt).
-  The minimum version required for now is [v0.8.0](https://github.com/coreos/rkt/releases/tag/v0.8.0).
+  The minimum version required is [v1.9.0](https://github.com/coreos/rkt/releases/tag/v1.9.0).
 
-- Note that for rkt version later than v0.7.0, `metadata service` is not required for running pods in private networks. So now rkt pods will not register the metadata service be default.
-
-- Since release [v1.2.0-alpha.5](https://github.com/kubernetes/kubernetes/releases/tag/v1.2.0-alpha.5),
-the [rkt API service](https://github.com/coreos/rkt/blob/master/api/v1alpha/README.md)
+The [rkt API service](http://coreos.com/rkt/docs/latest/subcommands/api-service.html)
 must be running on the node.
 
 ### Network Setup
 
-rkt uses the [Container Network Interface (CNI)](https://github.com/appc/cni)
-to manage container networking. By default, all pods attempt to join a network
-called `rkt.kubernetes.io`, which is currently defined [in
-`rkt.go`](https://github.com/kubernetes/kubernetes/blob/v1.2.0-alpha.6/pkg/kubelet/rkt/rkt.go#L91).
-In order for pods to get correct IP addresses, the CNI config file must be
-edited to add this `rkt.kubernetes.io` network:
+You can configure your network using the kubenet and CNI [network
+plugins](http://kubernetes.io/docs/admin/network-plugins/) by setting the
+`--network-plugin` flag, as described in each plugin's documentation.
+
+
+In addition, rkt supports using rkt's [Contained Networking](https://coreos.com/rkt/docs/latest/networking.html#contained-mode). In this mode, rkt will attempt to join pods into a network named `rkt.kubernetes.io`.
+
+However, there are a small number of caveats you should be aware of when using rkt's networking:
+
+* You must create an appropriate CNI configuration file with a network name of `rkt.kubernetes.io`.
+* The downwards API and environment variable substitution will not contain the pod IP.
+* The `/etc/hosts` file will not contain your own hostname.
```

Comment:
OK, nvm. `--hostname` flag of rkt modifies the `/etc/hostname` instead of `/etc/hosts`